src/SecurityDetails/SecurityDetailsOptions.ts

```diff
@@ -104,6 +104,7 @@ export type SecurityDetailsConverterType  = (
    | 'IndustryGroupBreakdown'
    | 'BondStatistics'
    | 'Meta'
+   | 'BondStyleBoxBreakdown'
```

Comment:
Just `BondStyleBox`. It is not a breakdown of grouped data, but rather the opposite: a grouping into specific categories.

Reply:
Oh, just realized this is the name of the property in the returned response.. Nevermind then, compatibility with response is more important.

Reply:
Yeah, I would stick to the props returned from API. Especially, that we have `BondStyleBox` in XRay and they have a different structure (you know)